### ISSUE TITLE
request to make joyride less sensitive to bad config

### ISSUE THREAD

**Issue by steve-tapley** [abhikmitra/ng-joyride#26]
Hi,
A couple of times I've set up joyride with misconfigured config (eg a bad/mistyped element name).   When the user happens to get to the step, joyride just stops working - the curtain is down, but the user has no way of recovering out of the joyride (unless they F5)

You can see this in the plunkr: http://plnkr.co/edit/Yn7cqL8IpR1VoL1ky65R

I could see this happening on a largish project where element ids change, and the tour is mistakenly not updated.

Can I make a suggestion that for misconfigured configs, that ng-joyride either end the tour, or skip the step?

Thanks,
Steve

**Comment by abhikmitra** [comment]
Makes sense. While it might be difficult to make it resilient from all sorts of bad config errors.It might make sense to skip for elements that are not present.

**Comment by tim-stefanski** [comment]
+1 for this enhancement. Skipping a missing element rather than getting stuck would be a great feature.

**Comment by learner972** [comment]
@abhikmitra  Could you please tell me how to include this as I am getting error if in case ID is not found
